core/templates/dev/head/css/oppia.css

```diff
@@ -1358,6 +1358,20 @@ pre.oppia-pre-wrapped-text {
   text-align: left;
 }
 
+.protractor-test-state-content-display > p {
```

Comment:
Make a new class for this purpose. The protractor-test classes are used only for the e2e testing.